extensions/wikia/ARecoveryEngine/js/helper.js

```diff
@@ -67,7 +67,7 @@ define('ext.wikia.aRecoveryEngine.recovery.helper', [
 					xmlHttp.send();
 				} catch (e) {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AEmptyBlock)